tests/lib/authentication.py

```diff
@@ -56,7 +58,6 @@ def generate_authentication(env):
             )
             details = json.loads(resp.content)
 
-            tokens[env] = (f"Bearer {details.get('access_token')}")
-            call_time = int(time())
+            self.tokens[env] = (f"Bearer {details.get('access_token')}", int(time()))

```

Comment:
It would be nice to give context as to what `int(time())` is meant to represent by creating a local var like was done previously (e.g. `call_time = int(time())`), but as mentioned I'm not sure I like `call_time`

Reply:
It's tied to the specific token, I don't really like the approach of using a global variable like it did before. I've put this in a tuple but if that's really an issue I think we can use namespaces or annotated tuples or dictionaries. Just seemed like overkill given it's so self contained.